src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -803,6 +818,21 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const renderNametagsToggle = () => (
+    <div className="toggle-room">
+      <h4 className="italic input-header">Display profile names on avatars</h4>
```

Comment:
[NB] Not a big thing one way or another, just musing if this wording would work better.

```suggestion
      <h4 className="italic input-header">Display user's names on their avatars</h4>
```

Reply:
fixed